pkg/kube/apisix/apis/config/v2beta3/types.go

```diff
@@ -338,11 +338,12 @@ type ApisixConsumerSpec struct {
 }
 
 type ApisixConsumerAuthParameter struct {
-	BasicAuth *ApisixConsumerBasicAuth `json:"basicAuth,omitempty" yaml:"basicAuth"`
-	KeyAuth   *ApisixConsumerKeyAuth   `json:"keyAuth,omitempty" yaml:"keyAuth"`
-	WolfRBAC  *ApisixConsumerWolfRBAC  `json:"wolfRBAC,omitempty" yaml:"wolfRBAC"`
-	JwtAuth   *ApisixConsumerJwtAuth   `json:"jwtAuth,omitempty" yaml:"jwtAuth"`
-	HMACAuth  *ApisixConsumerHMACAuth  `json:"hmacAuth,omitempty" yaml:"hmacAuth"`
+	BasicAuth     *ApisixConsumerBasicAuth     `json:"basicAuth,omitempty" yaml:"basicAuth"`
```

Comment:
v2beta3 has been deprecated and we don't need to add any functionality to it.

Reply:
done